onboarding/src/Components/CompanyInfo/CompanyInfoPreview.js

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import parse from "html-react-parser";
+
+function CompanyInfoPreview({ logo, link, companyName, mission, aboutCompany }) {
+  let videoLink;
+  if (
+    link.match(
+      /^(?:(?:(?:https?:)?\/\/)?(?:www\.)?(?:youtu(?:be\.com|\.be))\/(?:watch\?v\=|v\/|embed\/)?([\w\-]+))/i
+    )
+  ) {
+    videoLink = link.replace(/watch\?v=/g, "embed/").replace(/&[\w]+=[\w]+/g, "");
+  } else if (
+    link?.match(/^(?:(?:https?:\/\/)?(?:www\.)?vimeo\.com.*\/([\w\-]+))/i)
+  ) {
+    videoLink = link.replace(/vimeo\.com/g, "player.vimeo.com/video");
+  }
```

Comment:
It is repeated 3 times, so maybe we should think to move it into one function in `utils`
to try to make development 'solid'. (*)